src/components/editor/inputs/InputLang.jsx

```diff
@@ -112,13 +128,17 @@ const InputLang = (props) => {
             </div>
           </div>
           <div className="modal-footer">
+            <span className={alertClasses.join(" ")}>
+              Please select a valid language.
+            </span>
             <button className="btn btn-link" onClick={close}>
               Cancel
             </button>
             <button
               className="btn btn-primary"
               onClick={handleLangSubmit}
               data-testid={`submit-${props.textValue}`}
+              disabled={!submitEnabled}
```

Comment:
allow us to disable the submit button if the user is in a bad state, this will also trigger the alert above to show